semantic_conventions/lib/opentelemetry/semantic_conventions/common.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SemanticConventions
+    module Common
+      # A comparison between the actual size of the Span and the BatchSpanProcessor @max_queue_size
+      OTEL_BSP_BUFFER_UTILIZATION = 'otel.bsp.buffer_utilization'
```

Comment:
broadly this looks good, thank you for this work. i re-ran the windows job. a question that i'm not sure about is whether a few of these constants, like `buffer_utilization`, should live in the semantic conventions gem or whether it should be in the gem where the metrics reporter is being used, or maybe a different module like `Experimental` within this semconv gem, as `buffer_utilization` is not in the specification. don't think this is blocking tho, and i think this applies to only a few of the attributes in this pr. and i don't really mind if they end up under `Common` but just trying to give this a review, otherwise lgtm

Reply:
I'd vote for putting off-spec constants in a separate module within `OpenTelemetry::SemanticConventions`. `OpenTelemetry::SemanticConventions::OffSpec` or whatever, but this is a weak opinion, weakly held.

Reply:
sorry for the delay, yes i think we're on the same page, the consensus from the tuesday SIG(see https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit#heading=h.951oi3x8ptux)  was that these off spec constants should live in their gems, rather than common, so this should apply to the following metric families in this pr.

- `otel.jaeger_exporter.*`
- `otel.otlp_exporter.*`
- `otel.zipkin_exporter.*`
- `otel.bsp.*`

Reply:
Since I missed the SIG (again 😢) I presume that means we won't be taking this PR as-is @ericmustin?

(@Coolomina I appreciate your willingness to contribute and patience with us as we figure out where to put stuff like this. ❤️)

Reply:
@ahayworth yeah, the SIG and @Coolomina are in agreement on this plan:

> Now that I have a bit more context, and looking at the angles @plantfansam and @ericmustin provided, maybe it makes sense for off-spec attributes to live in their respective libraries.

Reply:
👍 Sounds fine with me.

Reply:
i mean we can take it and then just do a quick followup pr to move the stuff where we want it to be. i'm just a bit low on free cycles rn

Reply:
Hello! 

No need @ericmustin, I'm preparing a commit with the changes. It's not a problem at all @ahayworth :D, just trying to learn here.